File: Estructura de datos/Arrays Recorrer/ArraysRecorrer.cs
using System;

class Recorrer
{
    static void Main()
    {
        int[] datos = {10, 25, 37, 42, 58};

        Console.WriteLine("Recorrido del arreglo:");
        foreach (int val in datos)
        {
            Console.WriteLine(val);
        }
    }
}

File: Estructura de datos/Arrays Recorrer/ArraysRecorrer.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> datos = {10, 25, 37, 42, 58};

    cout << "Recorrido del arreglo:" << endl;
    for (int val : datos) {
        cout << val << endl;
    }

    return 0;
}
